### ISSUE TITLE
[bug] Individual wallet import TS error 

### ISSUE THREAD

**Issue by michaelxuwu** [rainbow-me/rainbowkit#788]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### RainbowKit Version

0.7.0

### wagmi Version

5.6.8

### Current Behavior

Importing individual wallets from the documented path `@rainbow-me/rainbowkit/wallets` generates the following TS error:

`Cannot find module '@rainbow-me/rainbowkit/wallets' or its corresponding type declarations.ts(2307)`

**Comment by amaurymagalhaes** [comment]
Same for me. I'm using these versions:
```json
"@rainbow-me/rainbowkit": "^0.7.0",
"next": "12.1.6",
"wagmi": "^0.6.7",
```

**Comment by amaurymagalhaes** [comment]
Seems like it have a bug since version [@rainbow-me/rainbowkit@0.7.0](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.7.0) when they changed from
```ts
import { connectorsForWallets, wallet } from '@rainbow-me/rainbowkit';

const wallets = [
  wallet.injected({ chains }),
  wallet.rainbow({ chains }),
  wallet.metaMask({ chains }),
  wallet.coinbase({ chains, appName: 'My App' }),
  wallet.walletConnect({ chains }),
];
```
to use as
```ts
import { connectorsForWallets } from '@rainbow-me/rainbowkit';
import { injectedWallet, rainbowWallet, metaMaskWallet, coinbaseWallet, walletConnectWallet, } from '@rainbow-me/rainbowkit/wallets';

const wallets = [
  injectedWallet({ chains }),
  rainbowWallet({ chains }),
  metaMaskWallet({ chains }),
  coinbaseWallet({ chains, appName: 'My App' }),
  walletConnectWallet({ chains }),
];
```

So if you downgrade to [@rainbow-me/rainbowkit@0.6.2](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.6.2), it gonna work.

**Comment by brightiron** [comment]
It appears that wallets is not exported correctly in the latest release. 
As a workaround you can import via:
```ts
import {
  metaMaskWallet,
  rainbowWallet,
  braveWallet,
  walletConnectWallet,
  coinbaseWallet,
} from "@rainbow-me/rainbowkit/dist/wallets/walletConnectors";
```
This should avoid ts warnings

**Comment by brightiron** [comment]
cc @markdalgleish . It looks like the exports from the package.json isn't working as intended with this latest release. I've confirmed it will introduce breaking changes unless you import as I referenced here above. 

Haven't had time to dig into the root cause, but did at least want to provide a temporary workaround so there's an upgrade path.

**Comment by markdalgleish** [comment]
This should be fixed now in [v0.7.1](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.7.1).
